src/components/settings/SettingsLMSTab/tests/CornerstoneConfig.test.jsx

```diff
@@ -53,7 +57,7 @@ describe('<CornerstoneConfig />', () => {
         enterpriseCustomerUuid={enterpriseId}
         onClick={mockOnClick}
         existingData={noExistingData}
-        existingConfigs={existingConfig}
```

Comment:
context why these were switched?

Reply:
This example isn't needed to be switched necessarily, I just want some examples where there are noConfigs, and I removed those from the ones that should have expected configs because they're included in the edit.